addons/portal/views/portal_templates.xml

```diff
@@ -310,7 +310,7 @@
                         </div>
                         <input type="text" class="form-control form-control-sm" placeholder="Search" t-att-value='search' name="search"/>
                         <span class="input-group-append">
-                            <button class="btn btn-secondary search-submit" type="button">
+                            <button class="btn btn-secondary" type="submit" t-attf-class="o_wait_lazy_js">
```

Comment:
no need of t-attf- here 🙂 You can just make:
```suggestion
                            <button class="btn btn-secondary" type="submit" class="o_wait_lazy_js">
```

Reply:
Maybe not a double class attribute? Neither suggestion would be working :grimacing:

Reply:
Ouch I did not see it 🙈 Thanks for your vigilance @qsm-odoo

Reply:
@cedricpri I updated the suggestion accordingly.